.devcontainer/Dockerfile

```diff
@@ -1,20 +1,40 @@
 ARG VARIANT="bullseye"
 FROM mcr.microsoft.com/vscode/devcontainers/rust:1-${VARIANT}
 
-RUN apt-get update
-RUN apt-get install -y \
-  git \
-  make
+# For installing just
+RUN <<EOF
+  wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
+  echo "deb [arch=all,$(dpkg --print-architecture) signed-by=/usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg] https://proget.makedeb.org prebuilt-mpr $(lsb_release -cs)" | tee /etc/apt/sources.list.d/prebuilt-mpr.list
+  apt update
+EOF
```

Comment:
Instead of this, can we use the curl alternative :
```
sudo curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -- --to /usr/bin
```

From https://github.com/casey/just?tab=readme-ov-file#pre-built-binaries

That way we won't have to depend on a third party repo

Otherwise LGTM